com/bl/ttt/tictactoe.py

```diff
@@ -32,27 +32,28 @@ def _is_cell_empty(self, move):
     def make_valid_move(self, move):
         self.cells[move.cell_chosen] = move.player_id
 
-    def is_any_row_complete(self, player_id):
+    def has_won(self, player_id):
+        if self._is_any_row_complete(player_id) or self._is_any_column_complete(player_id) or \
+                self._is_any_diagonal_complete(player_id):
+            return self
```

Comment:
Should be return true ?
also good to put in an else return false

Reply:
I know that None evaluates to False in truthiness but lets be explicit about it

Reply:
Updated.